pkg/kubelet/apis/cri/v1alpha1/runtime/api.proto

```diff
@@ -382,6 +384,9 @@ message PodSandboxStatus {
 message PodSandboxStatusResponse {
     // Status of the PodSandbox.
     PodSandboxStatus status = 1;
+    // Info is extra information of the PodSandbox in json format.
+    // It should only be returned non-empty when Verbose is true.
+    string info = 2;
```

Comment:
Should we also include comments about what should be in info for each response?

Reply:
Actually anything could be in info, I'll add several examples in the comment.

Reply:
Done